homeassistant/config_entries.py

```diff
@@ -1884,8 +1884,16 @@ async def async_forward_entry_setup(
             if not result:
                 return False
 
-        integration = await loader.async_get_integration(self.hass, domain)
+        if preload_platform:
```

Comment:
We still have integrations that call `async_forward_entry_setup` instead of `async_forward_entry_setups`